Src/Controller/AlunoController.hs

```diff
@@ -14,7 +20,31 @@ module Src.Controller.AlunoController where
         let aluno = Aluno matricula nome disciplinas
         adicionaLinha "Alunos" $ show aluno
         putStrLn "Aluno cadastrado com sucesso.\n"
-    
+
+
+    excluirTicket :: Int -> IO()
+    excluirTicket matAluno = do
+        putStrLn "Escolha entre os seus Tickets qual será excluido :"
+        ticketsIds <- pegaTicketsDoAluno matAluno
+        mostraTickets ticketsIds
```

Comment:
Faz mais sentido mostrar todos os tickets e depois pedir para ele dizer qual ticket ele quer excluir não? E depois mostrar uma mensagem confirmando essa deleção

Reply:
mas é justamente isso que ta fazendo ue

Reply:
To maluco